Source/WebCore/platform/mediastream/AudioMediaStreamTrackRenderer.h

```diff
@@ -134,9 +134,9 @@ inline const void* AudioMediaStreamTrackRenderer::logIdentifier() const
     return m_logIdentifier;
 }
 
-inline const char* AudioMediaStreamTrackRenderer::logClassName() const
+inline ASCIILiteral AudioMediaStreamTrackRenderer::logClassName() const
```

Comment:
Seems peculiar that we go out of our way to inline something that’s going to be called virtually.